### ISSUE TITLE
Left drawer doesn't change when a new organization language is selected

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa-admin#1925]
**Describe the bug**

The left drawer doesn't change when a new organization language is selected

**To Reproduce**

1. Select a new language for the organization
1. The left drawer doesn't update

**Expected behavior**

1. Select a new language for the organization
1. The left drawer updates
2. All tests must be valid and pass. They must test the fixing of this bug

**Actual behavior**

- See above

**Screenshots**

1. Select a new language for the organization

    ![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/16875803/21dd5d4b-6948-4216-b427-6e46b4b8f98a)

1. The left drawer doesn't update

    ![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/16875803/ca6ab0fe-6e36-459c-a50f-b05acbd2bbc2)

**Additional details**

- See above

**Potential internship candidates**

Please read this if you are planning to apply for a Palisadoes Foundation internship 
- https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by pranshugupta54** [comment]
Please assign this, I'm working on a related issue - https://github.com/PalisadoesFoundation/talawa-admin/issues/1579

**Comment by Cioppolo14** [comment]
I have assigned since the two issues are related.

**Comment by dhanagopu** [comment]
@pranshugupta54 Are you still working on this?

**Comment by pranshugupta54** [comment]
This is linked to another issue and we were planning to implement a full new better way.

**Comment by Cioppolo14** [comment]
@pranshugupta54 Just to clarify, is this issue on hold or is there a new issue with the better method we should keep open and close this?

**Comment by pranshugupta54** [comment]
@Cioppolo14, we're keeping this on hold for now. I was having conversation with Peter and we were kinda deciding about fixing translations completely. (Which will require huge changes)

**Comment by pranshugupta54** [comment]
We've kinda planned the timeline for this, I'll be making a PR soon for translations (this issue will be a part of it)

**Comment by pranshugupta54** [comment]
Working on this now, after #2002

**Comment by palisadoes** [comment]
@pranshugupta54 

We need to reconsider how we handle languages.

We have an organization level language setting, however this is insufficient if the end user wants to use a different language. We need to think of a way to handle per user language preferences. This will need to be implemented for both Admin and Mobile. Please create issues for this so we can track. We'll probably need ones for all three code repos to implement this.
